rm-datasource/src/main/java/com/alibaba/fescar/rm/datasource/ConnectionProxy.java

```diff
@@ -42,6 +43,8 @@
 
     private ConnectionContext context = new ConnectionContext();
 
+    private static int REPORT_RETRY_COUNT = ConfigurationFactory.getInstance().getInt(ConfigurationKeys.CLIENT_REPORT_RETRY_COUNT, 5);
```

Comment:
5 is magic value , use private static final DEFAULT_RETRY_COUNT

Reply:
done

Reply:
> done

need final

Reply:
ok